src/modals/taken/WidgetTaakForm.vue

```diff
@@ -342,13 +311,35 @@ export default {
 				})
 		},
 		fetchMedewerkers() {
-			this.medewerkersLoading = true
-			fetch('/index.php/apps/zaakafhandelapp/api/taken/medewerkers')
-				.then(response => response.json())
-				.then(data => {
-					console.log(data)
-					// this.medewerkers = data
-					// this.medewerkersLoading = false
+			medewerkerStore.refreshMedewerkersList()
+				.then(({ response, data }) => {
+
+					let selectedMedewerker = data.filter((medewerker) => medewerker?.id.toString() === taakStore.taakItem?.klant?.toString())[0] || null
+
+					if (this.taakId) {
+						selectedMedewerker = data.filter((medewerker) => medewerker?.id.toString() === this.taakItem.klant?.toString())[0] || null
```

Comment:
in plaats van `filter(...)[0]` kan je `find(...)` gebruiken.
maakt de code wat meer netjes